.vsts-ci/templates/ci-general.yml

```diff
@@ -11,6 +11,9 @@ steps:
 
       # Using `prependpath` to update the PATH just for this build.
       Write-Host "##vso[task.prependpath]$powerShellPath"
+
+      # Force override of unimportant errors in the install script
+      exit 0
```

Comment:
Already fixed this by having this step continueOnError. Please rebase from master